integration_tests/test_suites/daemon-test-suite/conftest.py

```diff
@@ -26,7 +28,7 @@ def foo_example_workspace_fixture(instance):
         PythonFileTarget(
             python_file=file_relative_path(__file__, "repo.py"),
             attribute=None,
-            working_directory=None,
+            working_directory=os.getcwd(),
```

Comment:
is it better to hard code this? this will vary if someone is locally executing pytest from a different directory not through tox. Not sure how likely that is, but explicit set maybe worth it.